src/graph/RadarTrackObjectsNode.cpp

```diff
@@ -54,6 +54,8 @@ void RadarTrackObjectsNode::enqueueExecImpl()
 	elevationHostPtr->copyFrom(input->getFieldData(ELEVATION_F32));
 	radialSpeedHostPtr->copyFrom(input->getFieldData(RADIAL_SPEED_F32));
 	entityIdHostPtr->copyFrom(input->getFieldData(ENTITY_ID_I32));
+	velocityRelHostPtr->copyFrom(input->getFieldData(RELATIVE_VELOCITY_VEC3_F32));
+	velocityAbsHostPtr->copyFrom(input->getFieldData(ABSOLUTE_VELOCITY_VEC3_F32));
```

Comment:
These do not seem to be marked as required through `getRequiredFieldList()` method - is this fine?